tests/func/experiments/test_show.py

```diff
@@ -176,84 +177,84 @@ def test_show_checkpoint_branch(
     "i_metrics,i_params,e_metrics,e_params,included,excluded",
     [
         (
-            "foo",
-            "foo",
+            "f",
```

Comment:
shorten the name of the parameters, because it is too long to collapse in outputs.

```
┏━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━┳━━━┳━━━━━━━━━━━━━━━━━┳━━━┓
┃ Experiment ┃ metrics.yaml:foo ┃ metrics.yaml:bar ┃ … ┃ params.yaml:foo ┃ … ┃
┡━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━╇━━━╇━━━━━━━━━━━━━━━━━╇━━━┩
│ workspace  │                1 │                1 │ 1 │ 1               │ 1 │
│ master     │                1 │                1 │ 1 │ 1               │ 1 │
└────────────┴──────────────────┴──────────────────┴───┴─────────────────┴───┘
```

Actually, in the previous test, we also have a column collapse in some of the test cases. But because there is a duplication in columns, we still get passed.

Reply:
This makes the test very hard to read, it's probably better to mock/patch the `Console` object used in `dvc.ui` to have a large enough width to fit the full table (instead of the default behavior where rich auto-detects however wide your current terminal session is).

Similar to how we override the width when we use the pager output:
https://github.com/iterative/dvc/blob/dba0aaeab7be3d369d1e41b6d7cd905c04e981af/dvc/ui/table.py#L57